packages/firestore/src/local/memory_persistence.ts

```diff
@@ -184,7 +184,9 @@ export interface MemoryReferenceDelegate extends ReferenceDelegate {
 }
 
 export class MemoryEagerDelegate implements MemoryReferenceDelegate {
-  private inMemoryPins: ReferenceSet | null = null;
+  /** Manages all documents that are active in Query views. */
+  private localViewReferences: ReferenceSet = new ReferenceSet();
+  /** The list of documents that are potentially GCed after each transaction. */
   private _orphanedDocuments: Set<DocumentKey> | null = null;
```

Comment:
Question: why is this variable prefixed with an underscore, while `localViewReferences` isn't?

Reply:
In general, we don't use underscore prefixes for classes that are not part of the private API. This is an exception as `orphanedDocuments` has a getter that verifies that is is set correctly. `orphanedDocuments` is cleared between each transaction run.